src/panels/alarm-table/renderer.js

```diff
@@ -259,7 +261,7 @@ export class TableRenderer {
       classesAsString = 'class="' + classes.join(' ') + '"';
     }
 
-    return '<td ' + stylesAsString + ' ' + classesAsString + '>' + value + widthHack + '</td>';
+    return '<td ' + stylesAsString + ' ' + classesAsString + title + '>' + value + widthHack + '</td>';
```

Comment:
Since sometimes the title will be empty, I just have the `' '` at the beginning when it's not, instead.  Then we don't end up with a spare space at the end of the tag.